spec/controllers/generals_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe GeneralsController, :type => :controller do
+	let(:user) {FactoryGirl.create(:user)}
+
+let(:general) {FactoryGirl.create(:general)}
+
+  it "should create topics" do
+    general_params = {
+      general: {
+        title: "Mystring",
+        description: "Mystring"
+        
+      }
+    }
+    session[:user_id]= user.id
+    post :create, general_params
+    expect(General.count).to eq 1
```

Comment:
this test is not correct, u should check like expect(General.count).to  change(Group, :count).by(1)